tide_site_restriction.module

```diff
@@ -133,6 +137,14 @@ function tide_site_restriction_views_pre_view(ViewExecutable $view, $display_id,
       }
     }
   }
+  // Excluding "Delete Content" option for editors SDPA-3781.
+  if ($view->id() == 'summary_contents_filters' && in_array('editor', $account->getRoles())) {
```

Comment:
similar to my previous comment, we don't want to hard code role names so adding a helper would be ... helpful :)

Reply:
@GROwen I have made the required changes and created separate permission for this.